KLR/Trace/Term.lean

```diff
@@ -676,7 +676,6 @@ def builtinEnv : List (Name × Term) := Id.run do
     let fn := .builtin name none
```

Comment:
Let's not merge this change until we are 100% sure that it's not breaking anything in NKI. One example of breaking change is that neuron_isa.py doesn't have nisa.dma_compute. Please coordinate with Yongwei Yuan.

Reply:
The change will break NKI; see NKIFE-506. Meanwhile, I am going through `ISA.lean` and `neuron_isa.py` to make sure APIs are consistent.